geoprocessing/src/modules/shapefiles/shapefiles.service.ts

```diff
@@ -0,0 +1,56 @@
+import { Injectable, Logger } from '@nestjs/common';
+
+import { createReadStream, rmdirSync } from 'fs';
+const mapshaper = require('mapshaper');
+const unzipper = require('unzipper');
+
+@Injectable()
+export class ShapeFileService {
+  private readonly logger: Logger = new Logger(ShapeFileService.name);
+  constructor() {}
+
+  private _unzipShapefile(fileInfo: Express.Multer.File): Promise<string> {
```

Comment:
1) `_` is not necessary as the function is already `private`. 
2) it actually unzips any file, not only `shapefiles` - consider name change
3) shouldn't this method return `path` to unzipped file?

I would also consider moving `unzip` logic to standalone service (as well as handling removal of leftovers)